shared/modules/testoval_module.py

```diff
@@ -17,14 +18,24 @@
 footer = '</oval_definitions>'
 ovalns = "{http://oval.mitre.org/XMLSchema/oval-definitions-5}"
 
+try:
+    from openscap import oscap_get_version
+    if oscap_get_version() < 1.2:
+        oval_version = 5.10
+    else:
+        oval_version = 5.11
+except ImportError:
+    oval_version = 5.10
```

Comment:
NACK. this brings a new requirement that we never check for while configuring. Believe it or not we never rely on openscap python bindings for anything in SSG so it seems a bit reckless to use them just to check oscap version. We know the oscap version from configure time, perhaps we can use that?

Reply:
Actually @mpreisler this code is already in the `testoval.py` script today. See https://github.com/OpenSCAP/scap-security-guide/pull/2424/files#diff-bb9a7e4d6f8f1b653e72514b5f01b7f3L201. It was just moved up in the code itself. This allows detection of what version of oscap supports OVAL version 5.11. In addition, testoval.py is not using CMake or configure time, so it will never get the OVAL version from configure time. Also, there is an argument to change the OVAL version in the script now.

Reply:
@mpreisler `testoval.py` is a OVAL writers validation tool that exists outside of CMake and the build system. It is for an OVAL writer to validate that the OVAL that was written or changed actually works as expected. I think that having it go through the build system would make the tool unwieldy to use. Personally, I like that it gets the highest level of supported OVAL version of the oscap tool, and if I don't want that version, I know have a way to set it with an option.